pyfilesync/__init__.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
Nie dodawaj nowych plików bo tracimy historię. Jak chcesz zmienić nazwę to robisz:

    git mv wawsync pyfilesync